test-data/unit/daemon.test

```diff
@@ -99,6 +99,37 @@ from mypy.plugin import Plugin
 class Dummy(Plugin): pass
 def plugin(version): return Dummy
 
+[case testDaemonRunRestartGlobs]
+-- Ensure dmypy is not restarted if the configuration doesn't change and it contains globs
+-- Note: Backslash path separator in output is replaced with forward slash so the same test succeeds on Windows as well
+$ dmypy run -- foo --follow-imports=error --python-version=3.6 | {python} -c 'import sys; sys.stdout.write(sys.stdin.read().replace("\\", "/"))'
```

Comment:
`'` isn't a quote character in windows command line parsing, I think.

I would ditch the pipe stuff, though and do the slash normalization in the test case driver. We have a `normalize_error_messages` function that fixes path slashes in test output that we use in most of our test drivers but don't seem to call in `testdaemon`.